hack/charts/training-jobs/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: sagemaker-k8s-trainingjob
 version: 0.1.0
-description: A Helm chart for deploying the SageMaker TrainingJob for Kubernetes.
+description: A Helm chart for deploying a SageMaker TrainingJob from Kubernetes.
```

Comment:
We should add `appVersion` to all, or none.

Reply:
To me it didn't make sense to have an "appVersion" on the jobs, since they don't necessarily correspond 1-to-1 with the controller. They will need to change if we make changes to the CRD specs, but I don't think this will be every release. Thoughts?

Reply:
Ah, that makes sense. If anything we should have an `appVersion` that corresponds to our API types (`v1`). That way when v2 comes around the switch is easier? Does the helm tool itself use these annotations, or are they for humans?